### PR TITLE
blog(embertimes#136): add writeup on embermap el. mod. forw. video

### PR DESCRIPTION
**Pull request by jayjayjpg**

### CODE REVIEW

File: source/2020-02-21-the-ember-times-issue-136.md [hunk 1]
```diff
@@ -76,13 +76,21 @@ Give Tomsweeper a try and leave your feedback in the [#games channel](https://di
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
+## [Learn about element modifier forwarding with Ember Map 🗺](https://embermap.com/topics/what-s-new-in-ember/forwarding-element-modifiers-with-attributes-3-11)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+Looking for another quick learning today and got **10 minutes** to spare?
+Then our friends from [Ember Map](https://embermap.com/) have just the right content for you!
```
Comment (jayjayjpg) [inline source/2020-02-21-the-ember-times-issue-136.md]:
```suggestion
Then our friends from [Ember Map](https://embermap.com/) have the right content for you!
```

File: source/2020-02-21-the-ember-times-issue-136.md [hunk 2]
```diff
@@ -76,13 +76,21 @@ Give Tomsweeper a try and leave your feedback in the [#games channel](https://di
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
+## [Learn about element modifier forwarding with Ember Map 🗺](https://embermap.com/topics/what-s-new-in-ember/forwarding-element-modifiers-with-attributes-3-11)
```
Comment (ijlee2) [inline source/2020-02-21-the-ember-times-issue-136.md]:
```suggestion
## [Learn about element modifier forwarding with EmberMap 🗺](https://embermap.com/topics/what-s-new-in-ember/forwarding-element-modifiers-with-attributes-3-11)
```

File: source/2020-02-21-the-ember-times-issue-136.md [hunk 3]
```diff
@@ -76,13 +76,21 @@ Give Tomsweeper a try and leave your feedback in the [#games channel](https://di
...
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+Looking for another quick learning today and got **10 minutes** to spare?
+Then our friends from [Ember Map](https://embermap.com/) have the right content for you!
```
Comment (ijlee2) [inline source/2020-02-21-the-ember-times-issue-136.md]:
```suggestion
Then our friends from [EmberMap](https://embermap.com/) have the right content for you!
```

File: source/2020-02-21-the-ember-times-issue-136.md [hunk 4]
```diff
@@ -76,13 +76,21 @@ Give Tomsweeper a try and leave your feedback in the [#games channel](https://di
...
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+In their freely available ["What's New in Ember?"](https://embermap.com/topics/what-s-new-in-ember) video series,
+they highlight some of the latest and greatest framework features that make Ember developer lives easier.
+
+In their latest episode ["Forwarding Element Modifiers with ...attributes"](https://embermap.com/topics/what-s-new-in-ember/forwarding-element-modifiers-with-attributes-3-11), you can learn about the use cases for the handy [splattributes feature](https://emberjs.github.io/rfcs/0311-angle-bracket-invocation.html#html-attributes) that made its way to Ember as part of [the RFC#311 anglebracket invocation feature](https://emberjs.github.io/rfcs/0311-angle-bracket-invocation.html).
```
Comment (ijlee2) [inline source/2020-02-21-the-ember-times-issue-136.md]: I think I'd like to see this paragraph in two sentences.
```suggestion
In their latest episode ["Forwarding Element Modifiers with ...attributes"](https://embermap.com/topics/what-s-new-in-ember/forwarding-element-modifiers-with-attributes-3-11), you can learn about the use cases for the handy [splattributes feature](https://emberjs.github.io/rfcs/0311-angle-bracket-invocation.html#html-attributes). Splattributes made their way to Ember as part of [the RFC#311 angle bracket invocation feature](https://emberjs.github.io/rfcs/0311-angle-bracket-invocation.html).
```

**Review by ijlee2 — APPROVED** [review]
Looks great, thank you for covering this!

File: source/2020-02-21-the-ember-times-issue-136.md (hunk 4 above)
Comment (jayjayjpg) [reply source/2020-02-21-the-ember-times-issue-136.md]: Yes, that sounds good, thank you for the suggestion!

**Comment by jayjayjpg** [comment]
🙇‍♀

**Pull request merged**